api/src/main/java/com/codeforcommunity/rest/FailureHandler.java

```diff
@@ -87,6 +88,13 @@ public void handleUserDoesNotExist(RoutingContext ctx, UserDoesNotExistException
     end(ctx, message, 400);
   }
 
+  public void handleUserDeleted(RoutingContext ctx, UserDeletedException e) {
+    String message =
```

Comment:
These changes include the changes from my previous PR... this will disappear once that gets merged.